content/source/docs/enterprise/run/install-software.html.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: "enterprise2"
+page_title: "Installing Software - Runs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-run-install"
+---
+
+# Installing Custom Software
+
+In some cases, it may be useful to install certain software on the Terraform worker, 
+such as a configuration management tool or cloud CLI.
+
+Terraform's `local-exec` [provisioner](https://www.terraform.io/docs/provisioners/local-exec.html) can be used in Terraform Enterprise (TFE). This will execute commands on the host machine running Terraform. In general, it should be used in conjunction with the resource(s) the software will be used to interact with, as relying on a placeholder (such as a null resource) may result in inconsistencies as to whether the software is installed.
```

Comment:
I suggest a wording change and leaving out the `null_resource` part until later, where it's explained that the container is destroyed, which is why it can't be used. Suggested changes in bold.

> Terraform's `local-exec` [provisioner](https://www.terraform.io/docs/provisioners/local-exec.html) *provisioner* can be used in Terraform Enterprise (TFE). This will execute commands on the host machine running Terraform. In general, *the provisioner configuration block containing the `local-exec` provisioner should be added to the resource(s) the software being installed* will be used to interact with.